src/Chat/Plugin/RFC.php

```diff
@@ -104,6 +205,9 @@ public function getHelpText(array $args): string
      */
     public function getCommandEndpoints(): array
     {
-        return [new PluginCommandEndpoint('Search', [$this, 'search'], 'rfcs')];
+        return [
+          new PluginCommandEndpoint('Search', [$this, 'search'], 'rfcs'),
+          new PluginCommandEndpoint('RFC', [$this, 'getRFC'], 'rfc'),
```

Comment:
For multiple endpoints it really needs a description in the 4th argument here for presentation reasons, but since the API is currently entirely undocumented I'll let you off :-D